spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,32 +4,61 @@
 context 'when switching phases' do
   subject { Season.current }
 
-  describe '#fake_application_phase' do
-    it 'timeshifts the application phase to today' do
-      Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
-        expect(subject).to be_application_period
+  describe 'destined' do
+
+    it 'sends the params to a phase setting method' do
+      phase = 'fake_application_phase'
+      Season::PhaseSwitcher.destined(phase)
+      expect(subject).to_not be_nil
+    end
+
+    context 'when directed to the #fake_application_phase' do
+      it 'sets date range so that the season is open for application submissions' do
+        phase = 'fake_application_phase'
+        Timecop.travel(Season.current.applications_close_at - 1.month) do
+          Season::PhaseSwitcher.destined(phase)
+          expect(subject).to_not be_nil
+          expect(subject).to be_application_period
+        end
       end
     end
-  end
 
-  describe '#fake_coding_phase' do
-    it 'timeshifts the coding phase to today' do
-      Timecop.travel(Season.current.ends_at - 1.week) do
-        Season::PhaseSwitcher.fake_coding_phase
-        expect(subject).to be_started
+    context 'when directed to the #fake_coding_phase' do
+      it 'sets dates so that the summer of code is currently happening' do
+        phase = 'fake_coding_phase'
+        Timecop.travel(Season.current.ends_at - 1.week) do
```

Comment:
Why is it necessary to do timetravel here *(this also applies to the following specs)*? The way I understand it, the underlying methods just update the certain values for the current `season`. So in theory it should be enough to do something along the lines of
```ruby
expect(Season)
  .to receive(:current)
  .and_return(season)
expect(season)
  .to receive(:update)
  .with(# all the things)
```
shouldn't it?

Besides that, I don't really understand why `Season.current` is the `subject` of your specs here. Isn't the thing that's being tested the `PhaseSwitcher`? So in my opinion, the `subject` should actually be
```ruby
described_class.destined(phase)
```
then you can of course also expect the changes on a season record with this. But I think this is outta scope for this test - the `Season` is already tested. I'd simply test if the respective methods get called on the season object.